airbyte-integrations/connectors/destination-streamr/destinations/streamr-destination/airbyte_secret/config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "privateKey": "9c9853f5ad682ae1552cf9bb205da37b704bc6d6105a94e190c5727d25e3e6cc",
+  "streamId": "0x0d0102474519cd2fc1b3e3f962a87e39cbcbead2/test-streamr"
```

Comment:
secrets must be placed inside the `secrets/` folder. @thanhlmm can you remove this?

Reply:
Solved. Thanks!